README.md

```diff
@@ -328,6 +328,38 @@ to match test name.
 => (#:greenlight.test{:description ",,,", :title "test-2", :ns user, :line 5, :steps []})
 ```
 
+### Parallel Test Execution
+
+Tests can be executed in parallel by providing a `--multithread` option
+with a number of threads. Tests can be further grouped with `::test/group`
+metadata to indicate that tests within the same group should run serially.
+
+If a `::test/group` is not provided, a test is placed in its own group.
```

Comment:
I really like this approach to the feature - it's a clever way of giving the user control without forcing serial tests to run after everything else.

Is there a way to select specific test groups to run? That could be a pretty nice feature for development as well as controlling some scenarios where you legitimately might want to pull out a test group for separate execution.

Reply:
I extended the `runner/find-tests` function to optionally take a map of test properties that can be matched again - namespace, description or this test group.